### PR TITLE
Indentation Changes

### PR DESCRIPTION
**Pull request by rachitchaurasia99**

### CODE REVIEW

File: app/models/cart.rb [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-        if current_item
-        current_item.quantity += 1
-        else
-        current_item = line_items.build(product_id: product.id)
-        end
-        current_item
-    end
-    def total_price
-        line_items.to_a.sum { |item| item.total_price }
+  has_many :line_items, dependent: :destroy
```
Comment (apekshit-tewathia) [inline app/models/cart.rb:2]: Add blank line between association and methods

File: app/models/order.rb [hunk 2]
```diff
@@ -1,51 +1,53 @@
...
-   enum pay_type: {
-"Check" => 0,
-"Credit card" => 1,
-"Purchase order" => 2
-}
- validates :name, :address, :email, presence: true
- validates :pay_type, inclusion: pay_types.keys
+  has_many :line_items, dependent: :destroy
+   
+  enum pay_type: {
```
Comment (apekshit-tewathia) [inline app/models/order.rb:3]: let's move enum above associations

File: app/models/product.rb [hunk 3]
```diff
@@ -3,10 +3,14 @@ class Product < ApplicationRecord
   validates :title, uniqueness: true
   validates :image_url, allow_blank: true, format: {with: %r{\.(gif|jpg|png)\z}i, message: 'must be a URL for GIF, JPG or PNG image.'}
   validates :price, numericality: { greater_than_or_equal_to: 0.01 }
+
   has_many :line_items
```
Comment (apekshit-tewathia) [inline app/models/product.rb]: let's follow the order
associations
validations
callbacks

**Review by apekshit-tewathia — CHANGES_REQUESTED** [review]

File: app/models/cart.rb (hunk 1 above)
Comment (rachitchaurasia99) [reply app/models/cart.rb:2]: done

File: app/models/order.rb (hunk 2 above)
Comment (rachitchaurasia99) [reply app/models/order.rb:3]: done

**Review by apekshit-tewathia — APPROVED** [review]

**Pull request merged**